notebooks/MAST/find_eclipse_candidates/find_eclipse_candidates.ipynb

```diff
@@ -0,0 +1,754 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    # JRK, I get an annoying units error from the table. Pretty sure it's Vizier's fault? 
</pre><p>I agree that this is either an issue with Vizier or the table itself. I think the problem is with [Fe/H] and [a/Fe] parameters which are given in solar units (written as "[Sun]" in the Vizier table). You might be able to silence Vizier warnings?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Just reran this on a different computer, and I don't get the error. I'll have to investigate if there's a version difference responsible for it.</p>

Reply:
<p>For what it's worth, I get the warning on my computer that is most updated. I removed the comment. The warning doesn't affect the workflow. </p>